src/py/reactpy/reactpy/backend/hooks.py

```diff
@@ -1,30 +0,0 @@
-from __future__ import annotations
```

Comment:
Oh - Actually one more thing. I'd like to release this change within the next minor rev (rather than immediately creating a major release).

To do that, we are going to need to keep the functions in `reactpy.backends.hooks` and provide a deprecation warning.

This deprecation also needs to be added in the changelog.